### PR TITLE
AUI floatpane drifting on Windows fixed

### PR DESCRIPTION
**Pull request by valid-ptr**
Solves #22533 issue

### CODE REVIEW

**Comment by vadz** [comment]
Looks good to me, thanks!

**Comment by acotty** [comment]
This is not in the 3.2.1 milestone page:
https://github.com/wxWidgets/wxWidgets/milestone/13

**Comment by vadz** [comment]
It has been already applied to 3.2 branch, see ec23938277c01835e9547e7453285dedaee5627b.

**Comment by acotty** [comment]
Sorry, I may not have been clear. Let me try again.

It's the milestone page that does not have any reference to the change in it, not the actual branch. So if someone looks at the milestone page without looking at the branch they will think the change is not included.  As such should the milestone page should include a reference to this or the commit?

**Comment by PBfordev** [comment]
> It's the milestone page that does not have any reference to the change in it

This is odd, I think I can see it there among [the closed issues](https://github.com/wxWidgets/wxWidgets/milestone/13?closed=1), third from the bottom?

**Comment by acotty** [comment]
I was looking at the open items. Thanks it is the third from the bottom in the closed list.
